$DragonFly: src/secure/usr.sbin/sshd/Attic/sshd.c.patch,v 1.3 2006/09/28 18:42:50 corecode Exp $
--- sshd.c	2006-09-17 06:04:46.000000000 +0200
+++ sshd.c	2006-09-28 19:58:34.000000000 +0200
@@ -80,6 +80,10 @@
 #include <prot.h>
 #endif
 
+#ifdef __DragonFly__
+#include <resolv.h>
+#endif
+
 #include "xmalloc.h"
 #include "ssh.h"
 #include "ssh1.h"
@@ -1630,6 +1634,17 @@
 	/* This is the child processing a new connection. */
 	setproctitle("%s", "[accepted]");
 
+#ifdef __DragonFly__
+       /*
+        * Initialize the resolver.  This may not happen automatically
+        * before privsep chroot().
+        */
+       if ((_res.options & RES_INIT) == 0) {
+               debug("res_init()");
+               res_init();
+       }
+#endif
+
 	/*
 	 * Create a new session and process group since the 4.4BSD
 	 * setlogin() affects the entire process group.  We don't
